driver-portal/src/ClientApp/src/app/letters-to-driver/letters-to-driver.component.ts

```diff
@@ -1,5 +1,3 @@
-// IMPORTANT keep this file identical to partner-portal letters-to-driver.component
-
```

Comment:
I would keep this until we move this to shared library, in case a new dev doesn't know that this code is duplicated